lib/cdo/rake_utils.rb

```diff
@@ -107,6 +107,7 @@ def self.system_stream_output(*args)
       error = RuntimeError.new("'#{command}' returned #{$?.exitstatus}")
       raise error, error.message
     end
+    0
```

Comment:
this makes `#system_stream_output` match the behavior of `#system` by returning the status code `0` on success.